examples/governor-bravo/BRAVO_03.sol

```diff
@@ -4,12 +4,23 @@ import {Vault} from "@examples/Vault.sol";
 import {MockToken} from "@examples/MockToken.sol";
 import {Addresses} from "@addresses/Addresses.sol";
 import {GovernorBravoProposal} from "@proposals/GovernorBravoProposal.sol";
+import {Proposal} from "@proposals/Proposal.sol";
 
 /// @notice Mock proposal that withdraws MockToken from Vault.
 contract BRAVO_03 is GovernorBravoProposal {
     /// @notice proposal name.
     string public override name = "BRAVO_03";
 
+    string private constant ADDRESSES_PATH = "./addresses/Addresses.json";
+
+    constructor()
+        Proposal(
+            ADDRESSES_PATH,
+            0x1a9C8182C09F50C8318d769245beA52c32BE35BC,
+            0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
```

Comment:
the build step caller and the deploy step deployer. I have changed the caller to the string name on the Addresses.json instead of the address. This ensures that even if the proposal or test changes the address (such as when doing etch running locally), the execution will always get the correct address